crates/landmass_rerecast/README.md

```diff
@@ -0,0 +1,153 @@
+# landmass_rerecast
+
+A plugin for [Bevy](https://bevyengine.org) to help using
+[bevy_landmass](https://github.com/andriyDev/landmass) and
+[bevy_rerecast_core](https://github.com/janhohenheim/rerecast) together.
```

Comment:
Most users should only care about `bevy_rerecast` sans core. If they don't need the editor stuff, the can disable the feature.

The only reason there's a split there in the first place is so that we can upstream them piecemeal :)